lib/html_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'sinatra/base'
+
+module Sinatra
```

Comment:
This helper was taken from the [official Sinatra documentation](http://www.sinatrarb.com/extensions.html#extending-the-request-context-with-sinatrahelpers) on how to add an HTML escape helper